src/app-layout/__integ__/app-layout-drawers.test.ts

```diff
@@ -126,6 +126,18 @@ for (const visualRefresh of [true, false]) {
       })
     );
 
+    test(
+      `should not shrink drawer beyond min width`,
+      setupTest({}, async page => {
+        // this width is small enough that it would hit the min-width of a drawer
+        await page.setWindowSize({ ...viewports.desktop, width: 700 });
```

Comment:
Screen size can go here in the config

```suggestion
      setupTest({ screenSize: { ...viewports.desktop, width: 700 } }, async page => {
```

The upside of this, it resizes the window before opening the page, so we do not need to worry if all resize listeners already worked or not

Reply:
I can approve after apply this change